cmd/s2i/main.go

```diff
@@ -203,7 +207,7 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().StringVar(&(oldScriptsFlag), "scripts", "", "Specify a URL for the assemble and run scripts")
 	buildCmd.Flags().StringVarP(&(oldDestination), "location", "l", "", "Specify a destination location for untar operation")
 	buildCmd.Flags().StringVarP(&(cfg.Destination), "destination", "d", "", "Specify a destination location for untar operation")
-	buildCmd.Flags().BoolVar(&(cfg.ForcePull), "force-pull", true, "Always pull the builder image even if it is present locally")
```

Comment:
ForcePull=true should set pull policy always, and ForcePull=false should set PullIfNotPresent.  

and we'll have to change the default, unfortunately, to ForcePull=false, otherwise it would conflict with the default of PINP for BuilderPullPolicy.